sources/api/schnauzer/src/helpers.rs

```diff
@@ -823,3 +876,62 @@ mod test_ecr_registry {
         assert_eq!(result, EXPECTED_URL_XY_ZTOWN_1);
     }
 }
+
+#[cfg(test)]
+mod test_host {
+    use super::*;
+    use handlebars::TemplateRenderError;
+    use serde::Serialize;
+    use serde_json::json;
+
+    // A thin wrapper around the handlebars render_template method that includes
+    // setup and registration of helpers
+    fn setup_and_render_template<T>(tmpl: &str, data: &T) -> Result<String, TemplateRenderError>
+    where
+        T: Serialize,
+    {
+        let mut registry = Handlebars::new();
+        registry.register_helper("host", Box::new(host));
+
+        registry.render_template(tmpl, data)
+    }
+
+    #[test]
+    fn not_absolute_url() {
```

Comment:
Apparently anything can be a valid URL as long as it has a scheme and the string does not contain any special characters, even if that scheme is something random. 

Going to opt for more stricter checking since otherwise anything can qualify as an URL host.